MetaData/python/samples_utils.py

```diff
@@ -984,7 +990,8 @@ def __call__(self):
         self.mn = SamplesManager("$CMSSW_BASE/src/%s/MetaData/data/%s/datasets*.json" % (options.metaDataSrc,options.campaign),
                                  dbs_instance=options.dbs_instance,
                                  force=options.doForce,
-                                 queue=options.queue,maxThreads=options.max_threads,doContinue=options.doContinue)
+                                 queue=options.queue,maxThreads=options.max_threads,doContinue=options.doContinue,
+                                 copyProxy=options.copy_proxy)
```

Comment:
@musella @michelif
This breaks all sample management tools. I made a quick fix with Nadya (removed copyProxy option) but I suspect that changes to other file were lost before making the PR

Reply:
The option should be defined. It was lsot ins some manupulation
```
make_option("--no-copy-proxy",dest="copy_proxy",action="store_false",
                            default=True,help="Do not try to copy the grid proxy to the worker nodes."
),
```